lib/utils/csvUpload.mjs

```
@@ -0,0 +1,194 @@
+/**
+## /utils/csvUpload
+
+This module exports the default csvUpload method for mapp utils module.
+
+@module /utils/csvUpload
+*/
+
+// Define schema methods.
+const schemaMethods = {
+  // eslint-disable-next-line quotes
+  Text: (val) => `'${val.replace(/'/g, "''")}'`,
+  Integer: (val) => parseInt(val.replace(/[^\d.-]/g, '')) || 'NULL',
+  Float: (val) => parseFloat(val.replace(/[^\d.-]/g, '')) || 'NULL',
+};
+
+/**
+@function csvUpload 
+@async
+
+@description
+This function uploads a CSV file to a database store.
+
+@param {File} file The CSV file to upload.
+@param {Object} params The parameters object.
+@property {String} params.query The query to upload the data.
+@property {Boolean} [params.async] If set to true, the data is uploaded asynchronously. Default is false.
+@property {Boolean} [params.header] If set to true, the first row is treated as a header row and not uploaded.
+
+@returns {<Promise>} The outcome object and any messages returned by the database.
+*/
+export default async function csvUpload(file, params = {}) {
+  if (!params.query) {
+    console.warn(`Query required`);
+    return;
+  }
```

Do you think we should throw an error here rather than just return??